target/linux/ath79/dts/qca9563_dlink_dir-869-a1.dts

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9563_dlink_dir-8x9-a1.dtsi"
+
```

Comment:
Please add `#include <dt-bindings/gpio/gpio.h>`, this file uses some GPIO contants